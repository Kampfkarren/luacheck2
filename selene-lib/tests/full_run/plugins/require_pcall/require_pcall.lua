cant_error()
can_error()

pcall(function()
	can_error()
end)

this.will_error()
