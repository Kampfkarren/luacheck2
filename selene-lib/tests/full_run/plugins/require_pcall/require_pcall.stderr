warning[plugin_require_pcall]: `can_error` must be wrapped in a pcall
  ┌─ require_pcall.lua:2:1
  │
2 │ can_error()
  │ ^^^^^^^^^

